### ISSUE TITLE
PHP Warning:  PHP Startup: Unable to load dynamic library

### ISSUE THREAD

**Issue by marktopper** [domainmod/domainmod#27]
When I run `php cron.php`, I get the following warning.
```
$ php cron.php
PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/lib/php/20160303/memcached.so' - /usr/lib/php/20160303/memcached.so: undefined symbol: php_msgpack_serialize in Unknown on line 0
```

**Comment by jennec** [comment]
Mark, assuming you are on a linux ditro, doing the following might help:

sudo apt-get install php-msgpack

And than enabling the php-msgpack with

sudo phpenmod msgpack

Restart apache after that.  

The above is something i read whilst trying to fix my domain queue issue #32, thought it might help.

**Comment by marktopper** [comment]
Thanks, this worked
